dictionary.txt

```diff
@@ -1,30 +1,86 @@
-# This sample dictionary for mfterm contains publicly available
```

Comment:
What's the point in changing this comment?

Reply:
I was copy my modified dictionary from [MifareClassicTool](http://github.com/ikarus23/MifareClassicTool/) project. I think its easier to keep updated only one version of dictionary.